src/signing/rsa-signature.ts

```diff
@@ -0,0 +1,47 @@
+import { pki, md } from 'node-forge';
+import { encodeSafe64, decodeSafe64 } from '../../src/util';
+
+
+export interface ISignature {
+    signature: string;
+    data: string;
+}
+
+export function signWithPrivateKey(privateKeyPem: string, data: string): ISignature {
+    const mdDigest = md.sha256.create();
+    const key = pki.privateKeyFromPem(privateKeyPem) as pki.rsa.PrivateKey;
+    mdDigest.update(data, 'utf8');
+    const signature = key.sign(mdDigest);
+    return {
+        signature: signature,
+        data: data
```

Comment:
Use shorthands for these

Reply:
@GenteiJanken
return {
        signature,
        data